scripts/projectView.js

```diff
@@ -1,23 +1,26 @@
-projectView = {};
+(function(module){
+  projectView = {};
 
-projectView.handleMainNav = function() {
-  $('.main-nav').on('click', 'li', function(){
-    var $content = $(this).attr('data-content');
-    $('.tab-content').hide();
-    $('.tab-content[id = "'+$content +'"]').fadeIn();
-  });
-        $('.main-nav .tab:first').click();
-};
+  projectView.handleMainNav = function() {
+    $('.main-nav').on('click', 'li', function(){
+      var $content = $(this).attr('data-content');
+      $('.tab-content').hide();
+      $('.tab-content[id = "'+$content +'"]').fadeIn();
+    });
+          $('.main-nav .tab:first').click();
+  };
```

Comment:
Might you refactor this so it doesn't require a click event to be called in order to display "home" at first? Would that be wise?